client/src/components/ReturnButton/ReturnButton.js

```diff
@@ -4,7 +4,7 @@ import PropTypes from "prop-types";
 
 const Button = styled.button`
   background: var(--color-white);
-  border: 1px solid var(--color-dark-border);
+  border: 0.1px solid var(--color-dark-border);
```

Comment:
Look the first comment. It's too small, bro.

Reply:
*That's what she said

Reply:
hahah thanks! you are right!